SwiftyJSONAccelerator/SJModelGenerator.swift

```diff
@@ -386,7 +386,9 @@ public class ModelGenerator {
 
     - returns: A single line mapping for the variable
     */
-    internal func mappingForObjectMapper(variableName: String, key: String) -> String {
+    internal func mappingForObjectMapper(variableName: String, var key: String) -> String {
+        key = key.stringByReplacingOccurrencesOfString("\"", withString: "", options: NSStringCompareOptions.LiteralSearch, range: nil)
```

Comment:
This change is not required. I understand your error - ` width <- map["kFBBackgroundUrlWidthKey"]` if you notice the constant variable is used as a string and it is wrong, but https://github.com/insanoid/SwiftyJSONAccelerator/pull/9 fixes the issue. However, if you are using the release then this change is not reflected in it (I will probably tag and create a release this weekend), you will have to build master for now.